### ISSUE TITLE
NTR: X-ray: maxillary decalcification (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8928]
New term request X-ray: maxillary decalcification (elthat_100829151834): Jaw; maxilla cystic lesions  UMLS: 3808877

**Comment by pnrobinson** [comment]
@MickeySegal
it seems that Maxillary cyst is the much more common designation? https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3415354/

**Comment by MickeySegal** [comment]
Decalcification is sometimes used https://www.omim.org/entry/615225 "she had palmoplantar hyperkeratosis, dyshidrosis, and maxillary decalcification with alveolysis and tooth loss".   But cyst may indeed be more common, so I'd be OK using that.

But note that we have another finding "jaw cystic lesions" elthat_100830172955 in the pipeline.

**Comment by pnrobinson** [comment]
@MickeySegal 
I still do not think that "Maxillary decalcification" is a good term label, since it is used to describe experimental procedures (e.g., "maxilla was decalcified by 5% nitric acid for 7 days") and the feature in OMIM:615225 was not shown in a figure nor was it precisely described. I agree that not all radiolucencies in the maxilla are related to cysts, but I am not sure how to define this particular feature. I will go ahead and make a term for Maxillary cyst, and we can figure out what to do about the rest when we have sufficient information

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000594
- label: Maxillary cyst
- parents: [Abnormal maxilla morphology (HP:0000326)](https://hpo.jax.org/app/browse/term/HP:0000326)
- pmid: 
- def: A closed sac that has developed in the upper jaw bone (maxilla) and contains fluid or semi-solid material. Cystic lesions of the maxilla are benign entities with both odontogenic and non-odontogenic origins.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8928
- synonyms:
